book/glossary.md

```diff
@@ -307,8 +307,8 @@ CSS properties (such as font sizing) are inherited by default.
 Style
 
 : All the pieces of information necessary to determine the visual
-display of an element. Also the name of a corresponding attribute to
-specify inline styles.
+display of an element. Also the name of the HTML element that
+specifies inline styles.
```

Comment:
I guess it is also an attribute, actually, now that I think about this. So maybe ignore this change.